### PR TITLE
Html css popup task

### PR DESCRIPTION
**Pull request by BlueLamantine**
Hello! This is my HTML & CSS practice
### Hooli-style Popup

[Demo](https://bluelamantine.github.io/kottans-frontend/task_html_css_popup/) |
[Code base](https://github.com/BlueLamantine/frontend-2021-homeworks/tree/html-css-popup-task/submissions/BlueLamantine/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/BlueLamantine/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="topnav">
+      <a href="#">Mail</a>
+      <a href="#">Images</a>
+
+      <input id="menu-toggle" type="checkbox" />
```
Comment (miroslav-kolomiets) [inline submissions/BlueLamantine/html-css-popup/index.html:19]: Good job! But something needs to be redone. For example, clicking on this place (see screenshot) opens a menu. This is mistake. And it is better to make the menu from the same elements.
Снимок экрана 2020-10-28 в 5 45 07 PM

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]

File: submissions/BlueLamantine/html-css-popup/index.html (hunk 1 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/html-css-popup/index.html:19]: Thanks for your attention to my work! Agree with you, my bad. I hid all elements-boxes from users on the page and now they are on the right places.

File: submissions/BlueLamantine/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <link rel="shortcut icon" href="img/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="topnav">
+      <a href="#">Mail</a>
+      <a href="#">Images</a>
```
Comment (madmaxWMFU) [inline submissions/BlueLamantine/html-css-popup/index.html:17]: Try to use more semantic tags.
```
<nav>
    <ul>
        <li>
            <a href="#"></a>
        </li>
        <li>
            <a href="#"></a>
        </li>
    </ul>
</nav>
```

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Nice job!
See comment bellow

File: submissions/BlueLamantine/html-css-popup/index.html (hunk 2 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/html-css-popup/index.html]: OK, why not! I rewrited html-stucture of page with semantic tags <ul><li>, all links nested into list items.

**Comment by madmaxWMFU** [comment]
Your button, not full width.

**Comment by BlueLamantine** [comment]
@miroslav-kolomiets  @madmaxWMFU  
Colleagues, all your comments have been incorporated in my practice work . Need I anything else to improve ?

**Review by madmaxWMFU — APPROVED** [review]

File: submissions/BlueLamantine/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    <nav class="topnav">
+      <ul class="topnav-list">
+        <li class="list-item">
+          <a href="#" class="nav-item">Mail</a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="nav-item">Images</a>
+        </li>
+        <li class="list-item">
+          <input id="menu-toggle" type="checkbox" />
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/html-css-popup/index.html:24]: Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about how to hide checkbox in the right way

File: submissions/BlueLamantine/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+          <label class="popup-btn nav-item" for="menu-toggle">
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/html-css-popup/index.html:25]: If you write code using BEM methodology in the current line you have a BEM element `popup-btn` without parent BEM block `popup`, If not, don't mind about this comment.

File: submissions/BlueLamantine/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+a:focus,
+.topnav input:focus + label {
+  background-color: var(--lightorange);
+  color: black;
+  outline: none;
+}
+.nav-item{
+  float: left;
+}
+#menu-toggle,
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/html-css-popup/style.css:52]: Avoid styling by `#id` and HTML tag, separate responsibilities:
- HTML tag for markup;
- CSS class for styling;
- id for JS logic, linking label with input

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments above!

File: submissions/BlueLamantine/html-css-popup/index.html (hunk 3 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/html-css-popup/index.html]: Thanks a lot for this useful information! This method has been implemented in my work.

File: submissions/BlueLamantine/html-css-popup/index.html (hunk 4 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/html-css-popup/index.html]: BEM methodology is welcomed, so I renamed alot of classes with BEM.

File: submissions/BlueLamantine/html-css-popup/style.css (hunk 5 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/html-css-popup/style.css]: It's a fair point, no more stylized by ```#id```, fixed it.

**Comment by BlueLamantine** [comment]
@A-Ostrovnyy :+1:
Thanks for interest to my homework. I followed your recomendations.

File: submissions/BlueLamantine/html-css-popup/style.css (hunk 5 above)
Comment (A-Ostrovnyy) [reply submissions/BlueLamantine/html-css-popup/style.css]: My point is if you can avoid styling by `#id` avoid it)

**Review by A-Ostrovnyy — APPROVED** [review]
LGTM!
Good work

**Review by miroslav-kolomiets — APPROVED** [review]
Good Job!

**Pull request merged**